setup.cfg

```diff
@@ -4,8 +4,10 @@ filterwarnings =
     ignore::DeprecationWarning
     ignore::PendingDeprecationWarning
 [flake8]
+max-line-length = 88
```

Comment:
This file is also relevant to changes :)

Reply:
Where does W503 come from? I can't see it in the link you shared.

Reply:
@freddyaboulton 
https://github.com/psf/black/issues/52 and https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator :)

Reply:
Thanks! Very cool!